Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import CoreLocation
 
 #if canImport(MapboxMapsFoundation)
```

Comment:
We should be able to remove these `canImport` statements now right?

Reply:
Yes, let's tackle as followup